src/extensions/queries.ts

```diff
@@ -18,3 +24,62 @@ export const installedApps = gql`
     }
   }
 `;
+
+export const installedAppsList = gql`
+  query InstalledAppsList(
+    $before: String
+    $after: String
+    $first: Int
+    $last: Int
+    $filter: AppFilterInput
+  ) {
+    apps(before: $before, after: $after, first: $first, last: $last, filter: $filter) {
+      pageInfo {
+        hasNextPage
+        hasPreviousPage
+        startCursor
+        endCursor
+      }
+      totalCount
+      edges {
+        node {
+          ...InstalledAppDetails
+        }
+      }
+    }
+  }
+`;
+
+export const eventDeliveries = gql`
+  query EventDelivery(
+    $before: String
+    $after: String
+    $first: Int
+    $last: Int
+    $filter: AppFilterInput
+    $canFetchAppEvents: Boolean!
+  ) {
+    apps(before: $before, after: $after, first: $first, last: $last, filter: $filter) {
+      pageInfo {
+        hasNextPage
+        hasPreviousPage
+        startCursor
+        endCursor
+      }
+      edges {
+        node {
+          id
+          ...AppEventDeliveries
+        }
+      }
+    }
+  }
+`;
```

Comment:
question: don't we already have a query for fetching event deliveries? (e.g. when displaying "!" icon on navigation)